rigex-front-end/src/components/navBar/navBar.scss

```diff
@@ -5,12 +5,12 @@
     background: $base-white;
 }
 
-.icon {
+.nav-icon {
     width: "30px";
     height: "30px";
 }
 
-.iconBox {
+.nav-icon-box {
```

Comment:
Let's begin using the power of scss by nesting our styles. Please do this on all scss files